sasdata/fair_database/data/models.py

```diff
@@ -87,6 +87,10 @@ class Quantity(models.Model):
 
     label = models.CharField(max_length=50)
 
+    dataset = models.ForeignKey(
+        DataSet, on_delete=models.CASCADE, related_name="data_contents"
+    )
+
```

Comment:
This is the right way to connect Quantity to DataSet and better than my suggestion :ok_hand: